products/idn/docs/identity-now/transforms/operations/lookup.md

```diff
@@ -55,10 +55,11 @@ transform involves a `table` entry of key-value pairs:
   - **table** - This is a JSON object of key-value pairs. The key is the string
     the transform tries to match to the input, and the value is the output
     string the transform returns if it matches the key.
-    > **Note** This is a use for the optional default key value here: if none of
-    > the three countries in the earlier example matches the input string, the
-    > transform returns "Unknown Region" for the attribute mapped to this
-    > transform.
+
+    :::tip
+    A default key must be specified otherwise an error will be returned
+    if there are no matching values in your table.
+    :::
 
```

Comment:
Can you change this to "A default key must be specified. Otherwise, an error..."